synchronization/src/cyface/java/de/cyface/synchronization/CyfaceAuthenticator.java

```diff
@@ -126,6 +147,32 @@ public Bundle getAuthToken(final @Nullable AccountAuthenticatorResponse response
         return bundle;
     }
 
+    /**
+     * Informs listeners, e.g. a SDK implementing app, about errors.
+     *
+     * @param context the {@link Context}
+     * @param errorCode the Cyface error code
+     * @param httpCode the HTTP error returned by the server
+     */
+    public static void sendErrorIntent(final Context context, final int errorCode, final int httpCode) {
```

Comment:
Maybe move this static method into ErrorHandler, so other classes may use it too.